README.md

```diff
@@ -3,16 +3,133 @@ Dagger Reflect
 
 A reflection-based implementation of the [Dagger][dagger] dependency injection library for fast IDE builds.
 
-More info soon...
+See [my talk (from about 37:25)][talk] for more details on how this came about.
+
+
+Er, what? Why would I want this?
+--------------------------------
+
+Feels like going back to Dagger 1 by Square? Not quite: we keep the benefits of Dagger 2, including the annotation processing compile-time validation in production, but using reflection speeds up development builds.
```

Comment:
Dagger 1 used code generation and compile-time validation. The only feature it didn't have compared Dagger 2 was... maps?

This project is more like Guice.